ui/core/wowhead.ts

```diff
@@ -142,3 +143,44 @@ export function getWowheadLanguagePrefix(): string {
 	const lang = getLang();
 	return lang === 'en' ? '' : `${lang}/`;
 }
+
+const WOWHEAD_SPEC_INDEX = {
+	[Spec.SpecBloodDeathKnight]: 0,
+	[Spec.SpecFrostDeathKnight]: 1,
+	[Spec.SpecUnholyDeathKnight]: 2,
+	[Spec.SpecBalanceDruid]: 0,
+	[Spec.SpecFeralDruid]: 1,
+	[Spec.SpecGuardianDruid]: 2,
+	[Spec.SpecRestorationDruid]: 3,
+	[Spec.SpecBeastMasteryHunter]: 0,
+	[Spec.SpecMarksmanshipHunter]: 1,
+	[Spec.SpecSurvivalHunter]: 2,
+	[Spec.SpecArcaneMage]: 0,
+	[Spec.SpecFireMage]: 1,
+	[Spec.SpecFrostMage]: 2,
+	[Spec.SpecBrewmasterMonk]: 0,
+	[Spec.SpecMistweaverMonk]: 1,
+	[Spec.SpecWindwalkerMonk]: 2,
+	[Spec.SpecHolyPaladin]: 0,
+	[Spec.SpecProtectionPaladin]: 1,
+	[Spec.SpecRetributionPaladin]: 2,
+	[Spec.SpecDisciplinePriest]: 0,
+	[Spec.SpecHolyPriest]: 1,
+	[Spec.SpecShadowPriest]: 2,
+	[Spec.SpecAssassinationRogue]: 0,
+	[Spec.SpecCombatRogue]: 1,
+	[Spec.SpecSubtletyRogue]: 2,
+	[Spec.SpecElementalShaman]: 0,
+	[Spec.SpecEnhancementShaman]: 1,
+	[Spec.SpecRestorationShaman]: 2,
+	[Spec.SpecAfflictionWarlock]: 0,
+	[Spec.SpecDemonologyWarlock]: 1,
+	[Spec.SpecDestructionWarlock]: 2,
+	[Spec.SpecArmsWarrior]: 0,
+	[Spec.SpecFuryWarrior]: 1,
+	[Spec.SpecProtectionWarrior]: 2,
+} as const;
+
```

Comment:
Are there specs that are different from the Spec Index we have on the spec TS classes?

<img width="505" height="109" alt="image" src="https://github.com/user-attachments/assets/4d9d50f9-4a1e-45d3-a89e-b266de322aec" />

Reply:
Thinking you might just be able to use `PlayerSpecs.fromProto`